packages/node/package.json

```diff
@@ -30,7 +29,13 @@
     "typescript": "^4.0.5"
   },
   "dependencies": {
-    "@inrupt/solid-client-authn-core": "^1.0.0"
+    "@inrupt/solid-client-authn-core": "^1.0.0",
+    "@types/uuid": "^8.3.0",
+    "cross-fetch": "^3.0.6",
```

Comment:
Since it only needs to work in Node, might as well just use `node-fetch` here?

Reply:
That was my initial intent, but the type definitions of `node-fetch` are a bit weird (`fetch` is both a function and a namespace, and it has an `isRedirected` property absent of the canonical `fetch`), and were creating inconsistencies that disappear when using `cross-fetch`.